pkgs/development/compilers/gcc/5/default.nix

```diff
@@ -547,4 +547,10 @@ stdenv.mkDerivation ({
 // optionalAttrs (!stripped || cross != null) { dontStrip = true; NIX_STRIP_DEBUG = 0; }
 
 // optionalAttrs (enableMultilib) { dontMoveLib64 = true; }
+
+# This is only here because it was here in previous commits and we
+# want to avoid a meaningless mass rebuild.  These variables get
+# overridden via crossAttrs in an actual cross-build.
+// { crossMingw = false; crossStageStatic = true; }
+
```

Comment:
So this is only required one time for the transition? If yes I think we should remove and deal with one mass-rebuild :wink: